MapboxNavigation/RouteTableViewController.swift

```diff
@@ -68,20 +68,25 @@ class RouteTableViewController: UIViewController {
 extension RouteTableViewController: UITableViewDelegate, UITableViewDataSource {
 
     func numberOfSections(in tableView: UITableView) -> Int {
-        return 1
+        return routeController.routeProgress.route.legs.count
+    }
+    
+    func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return "Leg \(section + 1)"
```

Comment:
Intermediate waypoints returned by the Directions API generally lack names. In that case, the second format is probably the best we can do, unless we’re OK with adding a dependency on the Geocoding API.